### DISCUSSION TITLE
theia command not found when running GLSP example

### DISCUSSION THREAD

**Discussion by gee4vee** [eclipse-glsp/glsp#576] [Q&A] [answered]
I'm trying to run the workflow example (glsp-client) but I'm getting:

```
/bin/sh: theia: command not found
```

Trying to figure out what I need from Theia so I cloned that repo and built it, but still nothing. How do I get the `theia` binary that the examples need?

**Comment by tortmayr** [comment, marked as answer]
Hi @gee4vee ,
I assume you are talking about the workflow example in the [glsp-examples](https://github.com/eclipse-glsp/glsp-examples) repository?
There should be no need to manually install any Theia dependencies. Everything you need should be provided as part of the dev dependencies.
However, I quickly checked and it seems like something got messed up with our start scripts in the root `package.json`. I prepared a fix for thatand now everything works as expected (following the instructions of the README).
Could you maybe check if it works for you if you checkout this [PR](https://github.com/eclipse-glsp/glsp-examples/pull/104)?

**Reply by gee4vee** [reply]
@tortmayr thanks for looking into it. I pulled latest from master but I get the same thing:

```
~/g/e/g/w/glsp-client [master]{1} ⛈️ »  yarn start                                                                                                   {09:32:05}
yarn run v1.22.17
$ yarn rebuild:browser && yarn --cwd workflow-browser-app start
$ theia rebuild:browser
/bin/sh: theia: command not found
```

I am having trouble building the server due to a Maven dependency issue. Is that why I don't have the `theia` binary?

**Reply by gee4vee** [reply]
Well I got my server build to succeed but still no `theia` binary.

**Reply by tortmayr** [reply]
That is strange. Could you maybe try if a `yarn install` executed in the repository root fixes the problem?

**Reply by gee4vee** [reply]
Looks like doing `yarn build:workflow` and `yarn start:workflow` from the repository root works for me. Thanks! My above error happens when executing within the `gsp-client` directory. Not sure if that's the right way of doing things.
